airflow/configuration.py

```diff
@@ -429,30 +428,34 @@ def _validate_enums(self):
                     )
 
     def _validate_config_dependencies(self):
-        """
-        Validate that config based on condition.
+        """Validate that config based on condition.
 
         Values are considered invalid when they conflict with other config values
         or system-level limitations and requirements.
         """
-        executor, _ = ExecutorLoader.import_default_executor_cls()
-        is_sqlite = "sqlite" in self.get("database", "sql_alchemy_conn")
+        self._validate_sqlite3_version()
```

Comment:
Rather than calling this from in here, call this function directly from The existing `validate()` function.